Source/Core/loadWithXhr.js

```diff
@@ -153,12 +153,20 @@ define([
         xhr.onload = function() {
             if (xhr.status >= 200 && xhr.status < 300) {
                 if (defined(xhr.response)) {
-                    deferred.resolve(xhr.response);
+                    if (responseType === 'json' && typeof xhr.response === 'string') { //IE doesn't support responseType = 'json'
```

Comment:
It's okay, but I would only do it for things that can't easily be done with actual XHR requests and make sure the behavior matches what browsers do.  For example, in all browsers, `responseType` defaults to an empty string and setting an unsupported type just sets `responseType` back to an empty string.